src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -221,12 +221,20 @@ internal partial class Session
 
             private Boolean IsAfterDot(Model model, ITextSnapshot textSnapshot, Dictionary<TextSpan, string> textSpanToText)
             {
-                var span = model.OriginalList.Span;
+                var originalSpan = model.OriginalList.Span;
 
                 // Move the span back one character if possible.
-                span = TextSpan.FromBounds(Math.Max(0, span.Start - 1), span.End);
+                var span = Span.FromBounds(Math.Max(0, originalSpan.Start - 1), originalSpan.End);
 
-                var text = model.GetCurrentTextInSnapshot(span, textSnapshot, textSpanToText);
+                // Because we are adjusting the span, it's not safe to call
+                // model.GetCurrentTextInSnapshot. GetCurrentTextInSnapshot starts by
+                // mapping the span into the ViewBuffer. In debugger intellisense, if the
```

Comment:
I don't think so. IsAfterDot really does want to know what appears in the subject buffer before the caret-in-subjectbuffer-position. In this case we're using TriggerSnapshot, which is the subject buffer, and the model's original span, which is also in terms of the subject buffer. No mapping should be required.